src/parser/SHOGunApplicationUtil.ts

```diff
@@ -396,7 +402,7 @@ class SHOGunApplicationUtil<T extends Application, S extends Layer> {
     olLayer.set('propertyConfig', layer.clientConfig?.propertyConfig);
     olLayer.set('legendUrl', layer.sourceConfig.legendUrl);
     olLayer.set('hoverable', layer.clientConfig?.hoverable);
-    olLayer.set('transparent', layer.sourceConfig?.transparent);
+    olLayer.set('transparent', layer.sourceConfig?.requestParams?.transparent);
```

Comment:
I think the `transparent` property is not needed to be set to the layer as it's already set as request parameter.